Gemfile

```diff
@@ -40,6 +40,7 @@ end
 
 gem 'net-ldap', '~> 0.10'
 gem 'omniauth-cas', '~> 1.1'
+gem 'omniauth-rails_csrf_protection'
```

Comment:
I have no experience with Ruby package management, but shouldn't we pin a specific version, just like in the linked guide?

Reply:
I'll pin the latest version then.